scenarios/centos-9/kuttl.yml

```diff
@@ -1,4 +1,6 @@
 ---
 cifmw_install_yamls_vars:
   STORAGE_CLASS: crc-csi-hostpath-provisioner
```

Comment:
Since we are enabling `make crc_storage` then we can drop ` STORAGE_CLASS: crc-csi-hostpath-provisioner`.